saleor/dashboard/category/views.py

```diff
@@ -22,7 +22,14 @@ def category_list(request):
     categories = get_paginator_items(
         category_filter.qs, settings.DASHBOARD_PAGINATE_BY,
         request.GET.get('page'))
-    ctx = {'categories': categories, 'filter': category_filter}
+    summary_msg = npgettext_lazy(
+        'Filter set results summary',
```

Comment:
Could we come up with a more descriptive context message here?

Reply:
Yes, I changed it and message relates now to what objects are filtered.